pyfar/utils.py

```diff
@@ -58,9 +61,29 @@ def broadcast_cshapes(signals, cshape=None):
     for signal in signals:
         if not isinstance(signal, (pf.Signal, pf.TimeData, pf.FrequencyData)):
             raise TypeError("All input data must be pyfar audio objects")
-
+    if cshape is not None and ignore_caxis is not None:
+        raise ValueError(f"Use ignore_caxis = 'None' to broadcast to cshape ="
+                         f" {cshape}.")
     if cshape is None:
-        cshape = np.broadcast_shapes(*[s.cshape for s in signals])
+        if ignore_caxis is not None:
+            data_shapes = [s.cshape for s in signals]
+            # Finds broadcast cshape without the axis to ignore
+            cshape = np.broadcast_shapes(*np.delete(data_shapes, ignore_caxis,
```

Comment:
Should we make sure that all cshapes have the same length before we do this by calling `pyfar.utils.match_cdims` beforehands? Otherwise the behavior would be hard to predict if signals with different `cdims` would be passed. If we do this we could add to the docstring of `caxis`: caxis refers to the channel axis after broadcasting the input signals to the same `cdim` by calling :py:func:`~pf.utils.broadcast_cdims`.